lms/djangoapps/django_comment_client/tests/group_id.py

```diff
@@ -95,7 +95,7 @@ def test_cohorted_topic_moderator_with_other_group_id(self, mock_request):
     def test_cohorted_topic_moderator_with_invalid_group_id(self, mock_request):
         invalid_id = self.student_cohort.id + self.moderator_cohort.id
         response = self.call_view(mock_request, "cohorted_topic", self.moderator, invalid_id)
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 500)
```

Comment:
After talking with Nimisha in the dev room, we agreed that the UI would use server error for all issues from the comment service. It was difficult to preserve the response codes from the comment service in the view that rendered the fragment, and it seemed incorrect. The fact that there was a bad request to the comment service, doesn't mean that the outer request itself should be considered bad. It seemed more accurate to say that this represents a server error.